Lang/es-es.json

```diff
@@ -61,7 +61,7 @@
 		"settingsLogEmuSha": "INFO - Versione di fpPS4: (%VARIABLE_0%)",
 		"dumpStatus_OK": "Bien",
 		"dumpStatus_WARN": "Archivos perdidos",
-		"dumpStatus_HB": "Cerveza casera"
+		"dumpStatus_HB": "Homebrew"
```

Comment:
Quick tip: If you need to make a string line get it's original value from English, you can leave it empty.
It's not something that you need to change, just a tip ;)